scalanet/src/io/iohk/scalanet/peergroup/kademlia/XorOrdering.scala

```diff
@@ -19,3 +21,26 @@ class XorOrdering(val base: BitVector) extends Ordering[BitVector] {
       0
   }
 }
+
+class XorNodeOrdering[A](val base: BitVector) extends Ordering[NodeRecord[A]] {
```

Comment:
Surely you can implement `XorNodeOrdering` in terms of `XordOrdering`? There seems to be a lot of copy-and-paste anyhow.

Reply:
`XorNodeOrdering` missing testcases.

Reply:
implemented in terms of `XordOrdering` so additional test cases are probably not that important now